packages/catalog-generator/README.md

```diff
@@ -7,11 +7,11 @@ This project is a simple tool to generate a Camel Catalog from a set of Camel co
 Install the project dependencies:
 
 ```bash
-mvn install
+mvnw install
```

Comment:
it is ./mvnw (at least on Linux)

Reply:
Ah right, otherwise we wouldn't be using the maven wrapper.